src/App.css

```diff
@@ -0,0 +1,48 @@
+.album-grid {
```

Comment:
I'm not seeing this used anywhere. I think you'll want to add this class to the `Grid` component in `AlbumGrid.js`: https://github.com/Code-the-Dream-School/dd-prac-team4-front/pull/17/files#diff-b4fb6f8095c236f3f4dc980d1c8dfaa22a66c02d184c7403c8d8f251515d92fbR9

Reply:
From testing this out; I think you'll also need to make the next line, `display: grid !important` to overtake the built-in Material UI flex-based grid styling. 

That will make the page go from this:

![](https://github.com/Code-the-Dream-School/dd-prac-team4-front/assets/9094098/cbf13784-5c24-43c1-b6c6-5f2e6b585f8f)

to this:

![](https://github.com/Code-the-Dream-School/dd-prac-team4-front/assets/9094098/cb1faa47-c9d5-4f21-a597-b67edb1f6fcd)